test/e2e/testsuites/mount.go

```diff
@@ -212,15 +219,19 @@ func (t *gcsFuseCSIMountTestSuite) DefineTests(driver storageframework.TestDrive
 	}
 
 	ginkgo.It("should pass --machine-type and --disable-autoconfig=false from driver to gcsfuse ", func() {
-		// TODO: remove this skip after we've updated the minimum sidecar version that supports the feature
-		e2eskipper.Skipf("skipping machine type defaulting test while new sidecar version is not known")
-		testDefaultingFlags()
+		if supportMachineTypeAutoconfig {
```

Comment:
Why are we relying on TEST_WITH_MACHINE_TYPE_AUTOCONFIG env variable to trigger  machine type tests. Can't they always be be on?